controller/backing_image_data_source_controller.go

```diff
@@ -978,7 +978,7 @@ func (m *BackingImageDataSourceMonitor) sync() {
 		if syncErr != nil {
 			m.retryCount++
 			if m.retryCount == engineapi.MaxMonitorRetryCount {
-				close(m.stopCh)
+				m.stopCh <- struct{}{}
```

Comment:
Also some `close(m.stopCh)` in the sync method.

Reply:
I have replace the all of closing channel behavior with sending `struct{}{}` into channel to notify stopping monitor.

Reply:
Have you reverified the fix?

Reply:
Yes, I verify the https://github.com/longhorn/longhorn/issues/4865 could be fixed and also running e2e test now.

Reply:
Update the e2e result for backingimage is pass
![image](https://user-images.githubusercontent.com/17548901/206347161-bff9b954-1dc9-4927-8d03-fefddd2aea4d.png)